spec/site_prism/page_objects/sections/cases/ico/ico_decision_section.rb

```diff
@@ -0,0 +1,17 @@
+module PageObjects
+  module Sections
+    module Cases
+      module ICO
+        class IcoDecisionSection < SitePrism::Section
```

Comment:
Can we keep this consistent and use `ICODecisionSection`?

Reply:
Good point